includes/admin/class-wcmypa-admin.php

```diff
@@ -979,8 +999,18 @@ public function showDeliveryDateForOrder(WC_Order $order): void
      */
     public function showShipmentConfirmation(WC_Order $order, bool $isEmail): void
     {
-        $deliveryOptions  = self::getDeliveryOptionsFromOrder($order);
-        $confirmationData = $this->getConfirmationData($deliveryOptions);
+        $deliveryOptions   = self::getDeliveryOptionsFromOrder($order);
+        $confirmationData  = $this->getConfirmationData($deliveryOptions);
+        $packageTypesArray = WCMYPA()->setting_collection->getByName(
```

Comment:
Is het zeker dat hier niet NULL of een andere waarde dan een array uit kan komen? Anders zou ik nog een lege array als default gebruiken hier.